indigo_linux_drivers/ccd_rpi/Makefile.inc

```diff
@@ -0,0 +1,30 @@
+ARCH := $(shell uname -m)
+OS := $(shell uname -s)
+
+ifeq ($(OS),Linux)
+  ifneq (,$(filter $(ARCH),aarch64 armv7l))
+    CCD_RPI := 1
+  endif
+endif
+
+ifeq ($(CCD_RPI), 1)
+
+SDK_DIR ?= $(wildcard bin_externals/*/lib/Linux/$(ARCH_DETECTED))
+
+SDK_INCLUDE ?= $(addprefix -I, $(wildcard bin_externals/*/include))
+SDK_LIB = -L$(SDK_DIR)
+SDK_LIB_A = $(SDK_DIR)/libcamera.a
```

Comment:
What version of libcamera have you imported ? By pulling the whole sources in - it's not clear what changes you have made to the libcamera project on top ...

There must be a cleaner way to express this - or build it as a component and represent any patches you have on top ?